.make/go.mk

```
@@ -64,13 +64,55 @@ go-mod-tidy:
 fmt:
 	gofmt -s -w $(SOURCES)
 
+#LINTERS += -E gofmt -E gci -E gofumpt -E goimports
+#LINTERS += -E gosec -E bodyclose -E dogsled -E durationcheck
+#LINTERS += -E revive -E ifshort -E makezero -E prealloc -E predeclared
+#LINTERS += -E asciicheck -E bidichk -E cyclop
+
+#Linters presets:
+#bugs: asciicheck, bidichk, bodyclose, contextcheck, durationcheck, errcheck, errorlint, exhaustive, exportloopref, gosec, govet, makezero, nilerr, noctx, rowserrcheck, scopelint, sqlclosecheck, staticcheck, typecheck
+#comment: godot, godox, misspell
+#complexity: cyclop, funlen, gocognit, gocyclo, nestif
+#error: errcheck, errorlint, goerr113, wrapcheck
+#format: gci, gofmt, gofumpt, goimports
+#import: depguard, gci, goimports, gomodguard
+#metalinter: gocritic, govet, revive, staticcheck
+#module: depguard, gomoddirectives, gomodguard
+#performance: bodyclose, maligned, noctx, prealloc
+#sql: rowserrcheck, sqlclosecheck
+#style: asciicheck, depguard, dogsled, dupl, errname, exhaustivestruct, forbidigo, forcetypeassert, gochecknoglobals, gochecknoinits, goconst, gocritic, godot, godox, goerr113, goheader, golint, gomnd, gomoddirectives, gomodguard, goprintffuncname, gosimple, ifshort, importas, interfacer, ireturn, lll, makezero, misspell, nakedret, nilnil, nlreturn, nolintlint, paralleltest, predeclared, promlinter, revive, stylecheck, tagliatelle, tenv, testpackage, thelper, tparallel, unconvert, varnamelen, wastedassign, whitespace, wrapcheck, wsl
+#test: exhaustivestruct, paralleltest, testpackage, tparallel
+#unused: deadcode, ineffassign, structcheck, unparam, unused, varcheck
+
+LINTERS += -p bugs
+LINTERS += -p comment
+LINTERS += -p error -D wrapcheck
+LINTERS += -p format
+LINTERS += -p complexity -D funlen -D cyclop
+LINTERS += -p import
+LINTERS += -p metalinter
+LINTERS += -p module
+LINTERS += -p performance
+LINTERS += -p sql
+LINTERS += -p style -D dupl -D goconst -D godox -D lll -D gomnd -D tagliatelle -D varnamelen -D ireturn -D nlreturn -D gochecknoglobals
+LINTERS += -p test -D exhaustivestruct
+LINTERS += -p unused
+LINTERS += -D interfacer # depreciated
```

```suggestion
LINTERS += -D interfacer # deprecated
```

Just a little note about these words: "depreciate/depreciated" is an accounting term for reducing the value of an asset.
"deprecate/deprecated" has become the technical term to use here. A bit of history https://www.merriam-webster.com/words-at-play/deprecate - it is not clear exactly why it started being used for this, but that is the common technical usage nowadays!